.github/CODEOWNERS

```diff
@@ -56,6 +56,6 @@
 /bin                                            @jrafanie @bdunne
 /content                                        @bdunne
 /config                                         @jrafanie
-/db                                             @agrare @carboni @bdunne
```

Comment:
Wow, there were several typos in the list

Reply:
Yeah I fixed any that jumped out at me.